docs/sections/user_guide/cli/mode_sfc_climo_gen.rst

```diff
@@ -0,0 +1,93 @@
+Mode ``sfc_climo_gen``
+======================
+
+The ``uw`` mode for configuring and running the :sfc-climo-gen:`sfc_climo_gen<>` component.
+
+.. code-block:: text
+
+   $ uw sfc_climo_gen --help
+   usage: uw sfc_climo_gen [-h] TASK ...
+   
+   Execute sfc_climo_gen tasks
+   
+   Optional arguments:
+     -h, --help
+         Show help and exit
+   
+   Positional arguments:
+     TASK
+       namelist_file
+         The namelist file
+       provisioned_run_directory
+         Run directory provisioned with all required content
+       run
+         Run execution
+       runscript
+         The runscript
```

Comment:
Info on the `mode_fv3.rst`:
```
    namelist_file
      The FV3 namelist file
    provisioned_run_directory
      The run directory provisioned with all required content
    run
      FV3 run execution
    runscript
      A runscript suitable for submission to the scheduler
```

Do we want to try to make the language the same here, i.e. Change `mode_fv3.rst`
```
    namelist_file
      The namelist file
    provisioned_run_directory
      Run directory provisioned with all required content
    run
      Run execution
    runscript
      A runscript suitable for submission to the scheduler
```

I think there was talk from moving away from specific instances of calling a driver name and making it more generic.

Reply:
Thank you for noticing this -- the code was actually updated to be more generic, but I failed to re-generate these output blocks. Jira ticket UW-491 will address automation of these blocks, so that they'll always be up-to-date, which will be nice. For now: Will fix (manually).